src/js/view/view.js

```diff
@@ -249,11 +249,21 @@ define([
             }
 
             _container = _playerElement.getElementsByClassName('jw-main')[0];
```

Comment:
@donato was `jw-main` removed? If so we need to create another container for plugins and a sweep needs to be done to cleanup orphaned code.